tools/i18n/update.js

```diff
@@ -69,22 +72,30 @@ htmlFiles.forEach(function (htmlFile) {
         return previous;
     }, {});
 
-    updateLanguageXlfJson(sourceHtmlRelativePath, strings).then(function (result) {
+    return updateLanguageXlfJson(sourceHtmlRelativePath, strings).then(function (result) {
```

Comment:
Sure, will fix. Though I hate the extra indentation that results. One day we'll move this to TypeScript and use await/async and avoid all this awkward Promise stuff 😄.